scripts/pancancer.py

```diff
@@ -0,0 +1,341 @@
+#! /usr/bin/python3
+import json
+import configparser
+import time
+import sys
+import subprocess
+import logging
+from cliff.app import App
```

Comment:
I typically don't like to import functions, or classes from a library
It might make your code cleaner to import cliff and reference the parts you need by the module.  
cliff.App.app
If this is recommended by the people who created cliff then disregard, it's a framework I've never seen.